index.js

```diff
@@ -32,7 +32,7 @@ function dispatchFromAgent (fn) {
     }
 
     const { origin, pathname, search } = util.parseURL(url)
-    const path = `${pathname || '/'}${search || ''}`
+    const path = `${pathname /* istanbul ignore next */ || '/'}${search || ''}`
```

Comment:
why ignoring? Could this even happen?

Reply:
This can't happen. Just remove the `||`.

Reply:
Done!